omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,155 +51,196 @@ running::
     $ javac -version
     javac 1.8.0_51
 
-Requirements
-------------
 
-.. _`Homebrew wiki`: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Installation.md
+OS X Basics
+-----------
 
-All the requirements for OMERO will be installed under :file:`/usr/local`. See also: Installation instructions on the `Homebrew wiki`_.
+In order to develop on OMERO, we recommend you ensure you have your Mac setup for
+development. The first step to achieving this is to create a :file:`.bash_profile` file in the
+root directory of your user folder.
 
-Install Homebrew and make sure :file:`/usr/local/bin` is prepended to your
-:envvar:`PATH`::
+To create a :file:`.bash_profile` from terminal, if one does not already exist::
 
-    $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-    $ export PATH=/usr/local/bin:$PATH
+    $ touch ~/.bash_profile
 
-Update Homebrew and run 'brew doctor' to fix potential issues beforehand::
+To open your :file:`.bash_profile` in a text editor, such as the built-in TextEdit app, use::
 
-    $ brew update
-    $ brew doctor
+    $ open -a TextEdit.app ~/.bash_profile
```

Comment:
this commands fails on snapper with 
```
open -a TextEdit.app ~/.bash_profile
LSOpenURLsWithRole() failed for the application /Applications/TextEdit.app with error -10810 for the file /opt/hudson/.bash_profile.
snapper:~ hudson$ 
```
maybe expected as I am ssh-ed in ? (cc @sbesson )

Reply:
@rgozim : I would leave the bit with TextEdit.app completely out. As we see, in some cases it does not work. In general, one should know how to edit a textfile in their system.

Reply:
Actually, backing up on my words. This line works fine on a normal Mac. It is just the ssh-ed in snapper which is throwing the error. Can leave as is.